build-logic/src/main/kotlin/publishing/util.kt

```diff
@@ -79,18 +79,6 @@ internal fun <T : Any> parseJson(url: String): T {
   }
 }
 
-internal fun fetchAsfProjectName(apacheId: String): String {
```

Comment:
Let's leave the function for now, I've got some stuff to adopt it.

Reply:
No problem for me (this function doesn't work today as the json returned is not the one expected), let me re-add it.

Reply:
The function is still there, I just added a `TODO` note to inform that it doesn't work for now.

Reply:
I'd suggest to remove it if it isn't used now. It won't be hard to reintroduce if needed.

Reply:
That was my initial change, but @snazy asked to keep the function for now (even if it doesn't work as I added ini the TODO comment).

@snazy what's your preference here ? keep the function or remove it for now ?

Reply:
Let's keep the function for now.

BTW: it's a habit in this project to add unused functionality as part of the overall development.

Reply:
OK, I keep the function with the `TODO` comment for now.

Reply:
To be clear the habit is to add independent testable and working code. This code doesn't work so the project norms don't apply.

Reply:
Yes, in order to find a consensus, I added the `TODO` comment to clean this up or fix in another PR (just to avoid to be stuck in this PR 😄 ).